projects/hca_mvp/CreateAdapterMetadata.wdl

```diff
@@ -0,0 +1,226 @@
+version 1.0
+
+import "../../projects/tasks/CreateOptimusAdapterObjects.wdl" as CreateOptimusObjects
+import "../../projects/tasks/MergeOptimusLooms.wdl" as MergeLooms
+import "../../projects/tasks/AdapterTasks.wdl" as Tasks
+import "../../projects/tasks/CreateReferenceMetadata.wdl" as CreateReferenceMetadata
+
+
+workflow CreateAdapterMetadata {
+  meta {
+    description: "Creates json objects for indexing HCA analysis data"
+    allowNestedInputs: true
+  }
+
```

Comment:
Maybe make a comments about fastqs being an array of space separated strings...our future selves will be thankful